cmd/cluster/encryption/encryptionutil.go

```diff
@@ -0,0 +1,100 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
+// agreements. See the NOTICE file distributed with this work for
+// additional information regarding copyright ownership. Yugabyte
+// licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package ear
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	ybmclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+	"golang.org/x/term"
+)
+
+func GetCmkSpecFromCommand(cmd *cobra.Command) (*ybmclient.CMKSpec, error) {
+	var cmkSpec *ybmclient.CMKSpec = nil
+	if cmd.Flags().Changed("encryption-spec") {
+		cmkString, _ := cmd.Flags().GetString("encryption-spec")
+		cmkProvider := ""
+		cmkAwsSecretKey := ""
+		cmkAwsAccessKey := ""
+		cmkAwsArnList := []string{}
+
+		for _, cmkInfo := range strings.Split(cmkString, ",") {
+			kvp := strings.Split(cmkInfo, "=")
+			if len(kvp) != 2 {
+				logrus.Fatalln("Incorrect format in cmk spec")
+			}
+			key := kvp[0]
+			val := kvp[1]
+			switch key {
+			case "provider":
```

Comment:
should be cloud-provider, we have rename everything in others PR

Reply:
Good point, thank you.